generated/python/grpc-google-service-logging-types-v1/setup.py

```diff
@@ -0,0 +1,43 @@
+"""A setup module for the GRPC Stackdriver Logging External Types service.
+
+See:
+https://packaging.python.org/en/latest/distributing.html
+https://github.com/pypa/sampleproject
+"""
+
+import setuptools
+
+from setuptools import setup, find_packages
+
+install_requires = [
+  'oauth2client>=2.0.0, <4.0dev',
+  'grpcio>=1.0.2, <2.0dev',
+  'googleapis-common-protos[grpc]>=1.5.0, <2.0dev'
+]
+
+setuptools.setup(
+  name='grpc-google-cloud-service-logging-types-v1',
+  version='0.14.0',
+  author='Google Inc',
+  author_email='googleapis-packages@google.com',
+  classifiers=[
+    'Intended Audience :: Developers',
+    'Development Status :: 3 - Alpha',
```

Comment:
Alpha is fine for now since we might get breaking feedback, but we probably want to declare Beta on this before long given that Logging itself is already in Beta.